db/views/organization_capacities_v01.sql

```diff
@@ -0,0 +1,20 @@
+WITH
```

Comment:
How come you need both the `schema.rb` change and this file? Is this basically the meat of the migration?

Reply:
I'm curious - does `rails db:rollback` work? If so, that's pretty cool.

Reply:
How does Scenic know it's v01 that you need to migrate to? Does it add a database table for tracking migrations on views? Asking earnestly.

Reply:
when you call rake scenic:migration something something, it creates this file and the migration file. Then we write the sql in here, and it does some magic to add it to the schema (i think that's where vita_partners_1 et al come from). YES! I did rollbacks and fowards several times! 

You can check out the gem documentation for all the cool stuff it can do -- you can create a new version of an existing view or update the same version directly. It also has support for materialized views.